pkg/sbom/v1/sbom_spdx_storage_format.go

```diff
@@ -22,39 +26,80 @@ const (
 	KubescapeOrganizationName = "Kubescape"
 	KubescapeNodeAgentName    = "KubescapeNodeAgent"
 	RelationshipContainType   = "CONTAINS"
+	directorySBOM             = "SBOM"
 )
 
+var spdxDataDirPath string
+
 type SBOMData struct {
-	spdxData                              spdxv1beta1.SBOMSPDXv2p3
+	spdxDataPath                          string
 	filteredSpdxData                      spdxv1beta1.SBOMSPDXv2p3Filtered
 	relevantRealtimeFilesBySPDXIdentifier sync.Map
 	newRelevantData                       bool
 	alreadyExistSBOM                      bool
+	instanceID                            instanceidhandler.IInstanceID
+}
+
+func init() {
+	wd, err := os.Getwd()
+	if err != nil {
+		logger.L().Ctx(context.GetBackgroundContext()).Error("failed to get working directory", helpers.Error(err))
+	}
+	spdxDataDirPath = fmt.Sprintf("%s/%s", wd, directorySBOM)
+	err = os.MkdirAll(spdxDataDirPath, os.ModeDir|os.ModePerm)
+	if err != nil {
```

Comment:
What will happen in such a case? Do we want the nodeAgent to panic?
Here you just print an error without doing anything.
If the program will continue running, then it is not an error.